synapse/rest/media/v1/upload_resource.py

```diff
@@ -101,6 +101,8 @@ async def _async_render_POST(self, request: SynapseRequest) -> None:
             # the default 404, as that would just be confusing.
             raise SynapseError(400, "Bad content")
 
-        logger.info("Uploaded content with URI %r", content_uri)
+        logger.info(f"Uploaded content with URI {content_uri}")
```

Comment:
I think we still prefer the %-style formatting when logging: it means the substitution only takes place if the logger is enabled at that level. (I doubt it makes a performance difference here, but it's nice to be consistent.)

Reply:
Oh, TIL!